tests/models/trillsson_efficientnet/test_modeling_trillsson_efficientnet.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch Trillsson_efficient model. """
+
+import unittest
+
+from datasets import load_dataset
+
+from transformers.testing_utils import is_torch_available, require_torch, slow, torch_device
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import TrillssonEfficientNetFeatureExtractor, TrillssonEfficientNetModel
+
+
+@require_torch
+@slow
+class TrillssonEfficientNetModelTester(unittest.TestCase):
```

Comment:
We really need integration tests where we compare the values from our model with the values we obtain from the TF version.

Reply:
I think it's a little bit hard to compare with TF version. In my local PC, I have to clone repo from google-research in [here](https://github.com/google-research/google-research.git) and init model like this [test](https://github.com/google-research/google-research/blob/a780b4f77374a933ca6aab0a9775a0efe7c0ce30/non_semantic_speech_benchmark/trillsson/models_test.py#L42-L49). So do you have any suggestion? 
I tried to fine-tune Trillsson model with Superb dataset for Keyword Spotting task, and the results look good, you can check in [here](https://huggingface.co/vumichien/trillsson3-ft-keyword-spotting-15)

Reply:
This should still be done 😓 We need users to be sure that out implementation matches 1-to-1 with the official code, otherwise they won't use it. 
So it will take a bit of time but it is valuable.